src/main/java/org/sonarlint/intellij/core/BackendService.kt

```diff
@@ -128,7 +140,16 @@ class BackendService @NonInjectable constructor(private val backend: SonarLintBa
                         this@BackendService.projectClosed(project)
                     }
                 })
-        }.thenApply { backend }
+        }.get()
```

Comment:
I thought at some point we tried to avoid that to not risk blocking the EDT, but I might be wrong